sql_queries/Populate_SpatialSubsetTable_auv.sql

```diff
@@ -0,0 +1,42 @@
+SET SEARCH_PATH = marvl3, public;
+	
+----AUV
+\echo 'AUV'
+INSERT INTO marvl3.spatial_subset(
+source_id,
+origin_id,
+"LONGITUDE",
+"LONGITUDE_QC",
+"LATITUDE",
+"LATITUDE_QC",
+"TIME",
+"TIME_QC",
+"DEPTH",
+"DEPTH_QC",
+"TEMP",
+"TEMP_QC",
+"PSAL",
+"PSAL_QC",
+geom
+)
+SELECT 
+s.source_id,
+d.file_id,
+d."LONGITUDE",
+'1',
+d."LATITUDE",
+'1',
+d."TIME" AT TIME ZONE'UTC',
+'1',
+d."DEPTH",
+"0",
```

Comment:
This triggers an error. Should be '0'. A string is declared with single quotes. Double quotes are only used to highlight the case of some objects like the "LONGITUDE" field.

Please check everywhere else and in other files.